tools/dependencies/known-oap-backend-dependencies.txt

```diff
@@ -61,6 +61,7 @@ httpclient-4.5.13.jar
 httpcore-4.4.13.jar
 httpcore-nio-4.4.13.jar
 influxdb-java-2.15.jar
+iotdb-session-0.12.2.jar
```

Comment:
How does IoTDB client implement the network requests? I can see there is no more dependency?

Reply:
Dependencies should be updated here, https://github.com/apache/skywalking/blob/fdbbd5a94be8c196fa69056e1bacdd05e1041f31/dist-material/release-docs/LICENSE#L222

Reply:
ok